vector/src/main/java/im/vector/riotx/features/home/room/detail/readreceipts/DisplayReadReceiptsBottomSheet.kt

```diff
@@ -67,6 +73,10 @@ class DisplayReadReceiptsBottomSheet : VectorBaseBottomSheetDialogFragment() {
         super.onDestroyView()
     }
 
+    override fun didSelectUser(senderId: String) {
```

Comment:
Please change `senderId` to `userId`. Also it can be renamed in OpenUserProfile() data class.